src/main/java/com/synopsys/integration/detect/tool/cache/InstalledToolManager.java

```diff
@@ -0,0 +1,34 @@
+/*
+ * synopsys-detect
+ *
+ * Copyright (c) 2021 Synopsys, Inc.
+ *
+ * Use subject to the terms and conditions of the Synopsys End User Software License and Maintenance Agreement. All rights reserved worldwide.
+ */
+package com.synopsys.integration.detect.tool.cache;
+
+import java.util.HashMap;
+import java.util.Map;
+
+public class InstalledToolManager {
+    public static final String INSTALLED_TOOL_FILE_NAME = "detect-installed-tools.json";
```

Comment:
This should be moved to `IntalledToolLocater` since that's the only place it's used.